src/Symfony/Bridge/Doctrine/Tests/DataCollector/DoctrineDataCollectorTest.php

```diff
@@ -249,7 +249,7 @@ private function createCollector($queries)
             ->method('getConnection')
             ->willReturn($connection);
 
-        $logger = $this->getMockBuilder(\Doctrine\DBAL\Logging\DebugStack::class)->getMock();
+        $logger = $this->createMock(\Doctrine\DBAL\Logging\DebugStack::class);
```

Comment:
I feel urged to replace the FQCN with an import, but apparently you could resist. 👏🏻 😅

Reply:
Will do, but the change right now is massive, lets check if tests are green and afterwards, I can replace the FQCN

Reply:
I think we should separate the steps. 🙈

Reply:
I think not, let's make it clean 😜